sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/BlobsCheckpointStore/BlobsCheckpointStoreLiveTests.cs

```diff
@@ -6,7 +6,6 @@
 using System.Linq;
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
-using Azure.Messaging.EventHubs.Processor.Tests.Infrastructure;
```

Comment:
I got a bit confused because with the `Infrastructure` namespace. It felt like we were not using it consistently (I might have just missed something). I removed it from every `Processor` test-related file. Should we keep this namespace?

Reply:
Nope;  I intended to remove it and seem to have missed a spot or two.   Visual Studio loves to add the folder name in by default.